src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -338,41 +317,16 @@ class WorkspaceInvitePage extends React.Component {
                                 )}
                             </View>
                             <View style={[styles.flexShrink0]}>
-                                <View style={[styles.ph5, styles.pv3]}>
-                                    <TextInput
-                                        label={this.props.translate('workspace.invite.personalMessagePrompt')}
-                                        autoCompleteType="off"
-                                        autoCorrect={false}
-                                        numberOfLines={4}
-                                        textAlignVertical="top"
-                                        multiline
-                                        containerStyles={[styles.workspaceInviteWelcome]}
-                                        value={this.state.welcomeNote}
-                                        onChangeText={text => this.setState({welcomeNote: text})}
-                                    />
-                                </View>
                                 <FormAlertWithSubmitButton
                                     isDisabled={!this.state.selectedOptions.length || this.state.shouldDisableButton}
                                     isAlertVisible={this.getShouldShowAlertPrompt()}
-                                    buttonText={this.props.translate('common.invite')}
+                                    buttonText={this.props.translate('common.next')}
                                     onSubmit={this.inviteUser}
                                     message={this.props.policy.alertMessage}
-                                    containerStyles={[styles.flexReset, styles.mb0, styles.flexGrow0, styles.flexShrink0, styles.flexBasisAuto]}
+                                    containerStyles={[styles.flexReset, styles.mb0, styles.flexGrow0, styles.flexShrink0, styles.flexBasisAuto, styles.mb3]}
```

Comment:
I see the margin-bottom from the design: `styles.mb5`. Also, we can remove that `styles.mb0`.

|Design|Current Change|
|---|---|
|<img src="https://user-images.githubusercontent.com/46707890/220824225-d526edd4-c1d4-4aae-9917-837a9f19de42.png" />|![Simulator Screen Shot - iPhone 14 - 2023-03-27 at 23 36 06](https://user-images.githubusercontent.com/25520267/228006788-42d54112-4c40-425e-9332-ced852fe9997.png)|